packages/svelte/src/tests/chat-synchronization.svelte

```diff
@@ -1,12 +1,13 @@
 <script lang="ts">
+  import { defaultChatStore } from '../chat-store.svelte.js';
   import { Chat } from '../chat.svelte.js';
   import { createAIContext } from '../context-provider.js';
 
   let { chatId }: { chatId?: string } = $props();
 
-  createAIContext();
-  const chat1 = new Chat({ chatId });
-  const chat2 = new Chat({ chatId });
+  createAIContext(defaultChatStore({ api: '/api/chat' }));
```

Comment:
@elliott-with-the-longest-name-on-github I believe we won't need this and the corresponding store context provider anymore, but want to confirm with you!